object.go

```diff
@@ -548,6 +548,17 @@ func (o *baseObject) setOwnStr(name unistring.String, val Value, throw bool) boo
 			o.val.runtime.typeErrorResult(throw, "Cannot add property %s, object is not extensible", name)
```

Comment:
[qq] I see that you added a nil check below for `o.val`, do we need it here as well?

Reply:
I was being overly cautious, I won't make changes here since this is from original goja. Thanks for checking though!